### PR TITLE
Revert "Merge pull request #23752 from MadmanMartian/from_hells_heart…

### PR DESCRIPTION
**Pull request by ghost**
*dab

:cl:
 * rscadd: The ability to dab has been re-added.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -210,5 +210,72 @@
...
+		H.visible_message("<span class='danger'>[H] holds one arm up and slams \his other arm into \his face! It looks like \he's trying to commit suicide.</span>",)
+		for(var/datum/organ/external/breakthis in H.get_organs(LIMB_LEFT_ARM, LIMB_RIGHT_ARM, LIMB_HEAD))
+			H.apply_damage(50, BRUTE, breakthis)
+			if(!(H.species.anatomy_flags & NO_BONES))
+				breakthis.fracture()
+		H.adjustOxyLoss(max(175 - H.getToxLoss() - H.getFireLoss() - H.getBruteLoss() - H.getOxyLoss(), 0))
+		H.updatehealth()
+	else
+		if(world.time-H.lastDab >= 10 SECONDS)
+			for(var/mob/living/M in view(0))
```
Comment (Kurfursten) [inline code/modules/mob/living/carbon/human/emote.dm]: this needs a center argument

**Comment by DrSnips** [comment]
Close. Just remove that april fools check and it's golden.

**Comment by PrimeDSS13** [comment]
my boy, it's him... it's been so long...

**Comment by ghost** [comment]
> 
> 
> Close. Just remove that april fools check and it's golden.

One step at a time.

**Comment by SonixApache** [comment]
delet

**Comment by ShiftyRail** [comment]
You really want to start a war? Just before christmas?

**Comment by DrSnips** [comment]
A reminder for those unable to attend dab in the fullest. You could dab at just ABOUT any time initally (i.e. in cryo). I deemed it a FEATURE after seeing it happen not even 1 round in.

**Comment by PrimeDSS13** [comment]
ehm collabs... where is the merge?

**Comment by SonixApache** [comment]
*why*

**Comment by anunknownentity03** [comment]
finally now I can bait people into dabbing and killing themselves again

**Pull request merged**